README.md

```diff
@@ -0,0 +1,121 @@
+![IPFS Logo] (ipfs.png)
+
+
+[![Join the chat at https://gitter.im/ipfs/py-ipfs](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ipfs/py-ipfs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+# py-ipfs
+
+Python implementation of IPFS, the InterPlanetary File System. Not even
+remotely done yet - check out [# fs1]{https://github.com/ipfs/py-ipfs/issues/1} to join the project.
+
+
+
+IPFS is a distributed file system that seeks to connect all computing devices
+with the same system of files. In some ways, this is similar to the original
+aims of the Web, but IPFS is actually more similar to a single bittorrent swarm
+exchanging git objects.
+
+IPFS could become a new major subsystem of the internet. If built right, it
+could complement or replace HTTP. It could complement or replace even more. It
+sounds crazy. It *is* crazy.
+
+Take a look at the main repo_.
+
+# Organization
+
+This repository contains the Python package `ipfs`, which contains the
+subpackages `block`, `merkledag`, `naming`, and `routing`, which function as
+laid out in the main IPFS repo_.
+
+.. _repo: http://github.com/ipfs/ipfs
+
+The repo roughly looks like this::
+    
+    ipfs
+    ├─ block
+    ├─ merkledag
+    ├─ naming
+    ├─ network
+    └─ routing
+
+# Contribute
+
+IPFS implementation in Python is a work in progress. As such, there's a few things you can do right now to help out:
+
+  * Go through the modules below and **check out existing issues**. This would be especially useful for modules in active development. Some knowledge of IPFS may be required, as well as the infrasture behind it - for instance, you may need to read up on p2p and more complex operations like muxing to be able to help technically.
+  * **Perform code reviews**. Most of this has been developed by @diasdavid, which means that more eyes will help a) speed the project along b) ensure quality and c) reduce possible future bugs.
```

Comment:
Yeah, but we should keep a reference to @diasdavid in that we're following his reimplementation in JS more than the golang one, which I sense is going to be most likely.